clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -77,7 +77,7 @@
 
   <xs:simpleType name="connectionUrl">
     <xs:restriction base="xs:anyURI">
-      <xs:pattern value="terracotta://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
+      <xs:pattern value="(terracotta|passthrough)://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
```

Comment:
I agree with you, @ljacomet but on the other hand I find it a bit extreme to have the URI scheme hardcoded in the XSD validation rule. IMHO the validation done in `ClusteringServiceConfiguration` is enough.